packages/core/package.json

```diff
@@ -24,12 +24,13 @@
     "typescript-standard": "0.3.36"
   },
   "dependencies": {
-    "wot-typescript-definitions": "0.7.3",
     "@node-wot/td-tools": "0.7.2",
-    "uuid": "3.4.0",
+    "@types/uritemplate": "0.3.4",
     "rxjs": "5.5.11",
     "uritemplate": "0.3.4",
-    "@types/uritemplate": "0.3.4"
+    "uuid": "3.4.0",
+    "vm2": "^3.9.2",
+    "wot-typescript-definitions": "0.7.3"
```

Comment:
FYI: This PR has the same changes as https://github.com/eclipse/thingweb.node-wot/pull/315

Reply:
thanks, I'll fix it and then this PR is ready to be merged

Reply:
Great, thanks! Please let us know once you think it is ready